test/katex-spec.js

```diff
@@ -1548,6 +1548,11 @@ describe("An HTML font tree-builder", function() {
         expect(markup).toContain(span);
     });
 
+    it("should render wide characters with mord and with the correct font", function() {
+        const markup = katex.renderToString(String.fromCharCode(0xD835, 0xDC00));
+        expect(markup).toContain("<span class=\"mord mathbf\">A</span>");
+    });
```

Comment:
We might also want a test like
```
expect("M = a + b").toBuildLike("\textbf{M = a + b}");
```
where "M = a + b" is the bolded versions.  Not sure the best way to store a string with bolded characters.

Reply:
I just tried something equivalent to:
```
expect("𝐀 = 𝐚").toBuildLike("\\mathbf{A} = \\mathbf{a}");
```

It turns out that they don't build in an equivalent way. `\mathbf{A}` returns a `mord` span wrapped inside a `mord mathbf` span. "𝐀" returns a single `mord mathbf` span. They render in an equivalent way, but the test doesn't like it.

Reply:
Can you test
```js
expect("𝐀 = 𝐚").toBuildLike("\\mathbf A = \\mathbf a");
```
?

Reply:
@edemaine That one works! Cool!